src/app/services/http.service.ts

```diff
@@ -18,21 +15,23 @@ export class HttpService {
   //   })
   // };
 
-  constructor(private http: HttpClient) { }
+  constructor(private http: HttpClient) {}
 
   // // use this when making get requests with params
   getRequestWithParams = (endpoint, params = {}) => {
     return this.http.get(this.base_url + endpoint, { params });
+  };
+  deleteRequestWithParams(endpoint, params = {}) {
+    return this.http.delete(this.base_url + endpoint, { params });
   }
 
   // use this for POST,PUT or PATCH.
   makeRequestWithData = (
     endpoint: any,
     data: any,
     method: any,
-    params?: any,
+    params?: any
   ) => {
     return this.http[method](`${this.base_url}${endpoint}`, data, params);
-  }
-
+  };
```

Comment:
The semicolon here is not necessary since semicolons are not required after block statements or function declarations. In this case, we have a function declared
```suggestion
  }
```